apps/wfo-ui/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,39 @@
+import NextAuth, { AuthOptions, Session } from 'next-auth';
+import KeycloakProvider from 'next-auth/providers/keycloak';
+import { JWT } from 'next-auth/jwt';
+
+export type SessionToken = Session & {
+    accessToken?: unknown;
+};
+
+export const authOptions: AuthOptions = {
+    providers: [
+        KeycloakProvider({
```

Comment:
Change to a custom Oauth provider https://next-auth.js.org/configuration/providers/oauth#using-a-custom-provider